### PR TITLE
Add Travis CI file to release binaries

### LINKED ISSUE #3: Generate releases
Need to build and publish release and tag. Blocked by #2.

### PR DESCRIPTION
**Pull request by dixonwille**
Fixes #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+# calls goreleaser
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: curl -sL https://git.io/goreleaser | bash
+    on:
+      tags: true
+      # it's important to build only on one item in matrix
+      condition: $TRAVIS_GO_VERSION =~ ^1\.13
```
Comment (komish) [inline .travis.yml]: Comment this section.

**Review by komish — CHANGES_REQUESTED** [review]
@dixonwille GoReleaser looks slick! For now, I'll ask that we comment out the travis goreleaser bits until I have more time to tinker with it. Ready to merge with these changes in place.

**Comment by dixonwille** [comment]
@komish Glad you like it. Done my fair share of make files for work. I usually only have to build the item for one OS and Arch though. Goreleaser is overkill for those scenarios.

I was also lucky enough to get in the beta and highly recommend looking at https://github.com/features/actions when it gets released. It has made CI so much nicer as it keeps everything in GitHub.

**Review by komish — APPROVED** [review]
Approved, LGTM!

**Comment by komish** [comment]
> I was also lucky enough to get in the beta and highly recommend looking at https://github.com/features/actions when it gets released. It has made CI so much nicer as it keeps everything in GitHub.

Good to know, I'll check it out as well! Thanks again @dixonwille

**Pull request merged**